src/lib.rs

```diff
@@ -362,12 +362,34 @@ impl std::fmt::Display for Language {
     }
 }
 
+#[derive(Debug)]
+pub struct ParseLanguageError(String);
+
+impl std::fmt::Display for ParseLanguageError {
```

Comment:
Style nits: I'd probably prefer to import the `std` items at the top level.

Reply:
Thank you for reviewing the PR.
In general, I tend to agree with you on this point. But in this case, the full path is also used at other places in this library, e.g. in [line 343](https://github.com/humenda/isolang-rs/blob/master/src/lib.rs#L343). Should they be changed too? What about `std::fmt::Result`, which has a very generic name and would conflict with `Result`?

Reply:
I usually like to import `std::fmt` and pull `fmt::Display`, `fmt::Formatter<'_>` and `fmt::Result` from there.

Reply:
I followed @humenda's suggestion and kept `std::fmt::Result` in place